doc/parameter.md

```diff
@@ -74,6 +75,28 @@ Parameters for Tree Booster
 * scale_pos_weight, [default=0]
   - Control the balance of positive and negative weights, useful for unbalanced classes. A typical value to consider: sum(negative  cases) / sum(positive cases) See [Parameters Tuning](param_tuning.md) for more discussion. Also see Higgs Kaggle competition demo for examples: [R](../demo/kaggle-higgs/higgs-train.R ), [py1](../demo/kaggle-higgs/higgs-numpy.py ), [py2](../demo/kaggle-higgs/higgs-cv.py ), [py3](../demo/guide-python/cross_validation.py) 
 
+Additional parameters for Dart
+------------------------------
+* samp_drop [default=0]
+  - type of sampling algorithm.
+    - 0: dropped trees are selected uniformly.
+    - 1: dropped trees are selected in proportion to weight.
+* norm_drop [default=0]
```

Comment:
use string(intuitive name) instead of integer for type for the enumeration.

see https://github.com/dmlc/xgboost/blob/master/src/tree/param.h#L74